src/main/java/io/fabric8/maven/docker/config/ImageConfiguration.java

```diff
@@ -33,6 +33,9 @@
     @Parameter
     private String registry;
 
+    @Parameter
+    private String file;
```

Comment:
Actually I would move this to the `<build>` configuration as it is about *creating images*. There is should be probably called something like `dockerArchive` on the same footprint as `dockerFile` or `dockerFileDir`. Also there should be a check, that only one of `dockerArchive` and `dockerFile` is given.